.github/workflows/build_packages.yaml

```diff
@@ -216,9 +217,9 @@ jobs:
           - debian11
           - debian10
           - debian9
-          - rockylinux8
-          - centos7
-          - raspbian10
+          - el8
+          - el7
+          # - raspbian10 #armv6l is too slow to emulate
```

Comment:
I don't know whether it should be commented out, it just synchronizes the changes made in v4.3.15-rc.3.

Reply:
yes, we can skip it all together for now.